.eslintrc.json

```diff
@@ -1,6 +1,8 @@
 {
   "extends": "@darkskyapp",
   "rules": {
-    "max-len": "off"
+    "max-len": "off",
+    "no-multi-spaces": "off",
+    "indent": "off"
```

Comment:
anybody know how I can allow multi spaces and indents just before numbers? because that'd be the best